src/analyze/analyze-condition.c

```diff
@@ -15,8 +15,9 @@ typedef struct condition_definition {
 } condition_definition;
 
 static const condition_definition condition_definitions[] = {
-        { "ConditionPathExists",             config_parse_unit_condition_path,   CONDITION_PATH_EXISTS              },
+        /* ConditionPathExistsGlob must be listed earlier than CondtionPathExists. */
```

Comment:
I was confused about why this works, but I see it's because the code uses `startswith()` to compare the condition name. Should we instead change it to use `streq()` so the ordering doesn't matter?

Reply:
Or do something like
```C
bool startswith_word(const char *line, const char *word) {
      const char *p = startswith(line, word);
      if (!p)
           return false;
      return strchr(" \t=", *p);
}
```
?